.github/CODEOWNERS

```diff
@@ -31,3 +31,21 @@ Dockerfile @crossedfall
 
 /code/game/gamemodes/dynamic @itsmeow
 /code/modules/modular_computers @itsmeow
+/tools/MapTileAggregator.py @itsmeow
```

Comment:
I'm not sure who actually gets code owner here. If it would give it to you for being a specific file or me for owning the directory. I think you might need to add it to the end as a multiple owners thing?

Reply:
Not sure

Reply:
I think it would consider us both as owning it